### PR TITLE
Added documentation changes for powerflex version 2.4.0

### PR DESCRIPTION
**Pull request by meenakshidembi691**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
Add support for PowerFlex ansible modules and roles on Azure.

| GitHub Issue # |
| -------------- |
| N/A |

# Checklist:

- [ X ] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [ X ] I have performed Ansible Sanity test using --docker default
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ X ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ X ] Backward compatibility is not broken

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,59 +6,59 @@ The capabilities of the Ansible modules are managing SDCs, volumes, snapshots, s
...
+The Ansible collection for PowerFlex is released and licensed under the GPL-3.0 license. See [LICENSE](https://github.com/dell/ansible-powerflex/blob/2.4.0/LICENSE) for the full terms. Ansible modules and modules utilities that are part of the Ansible collection for PowerFlex are released and licensed under the Apache 2.0 license. See [MODULE-LICENSE](https://github.com/dell/ansible-powerflex/blob/2.4.0/MODULE-LICENSE) for the full terms.
 
 ## Prerequisites
 
 | **Ansible Modules** | **PowerFlex/VxFlex OS Version** | **SDK version** | **Python version** | **Ansible**              |
 |---------------------|-----------------------|-------|--------------------|--------------------------|
-| v2.3.0 |3.6 <br> 4.0 <br> 4.5 | 1.10.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 |
+| v2.4.0 |3.6 <br> 4.0 <br> 4.5 | 1.10.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 <br> 2.17 <br> 2.18 |
```
Comment (Jennifer-John) [inline README.md:31]:
```suggestion
| v2.4.0 |3.6 <br> 4.0 <br> 4.5 | 1.10.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 |
```

**Review by Jennifer-John — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,59 +6,59 @@ The capabilities of the Ansible modules are managing SDCs, volumes, snapshots, s
...
+The Ansible collection for PowerFlex is released and licensed under the GPL-3.0 license. See [LICENSE](https://github.com/dell/ansible-powerflex/blob/2.4.0/LICENSE) for the full terms. Ansible modules and modules utilities that are part of the Ansible collection for PowerFlex are released and licensed under the Apache 2.0 license. See [MODULE-LICENSE](https://github.com/dell/ansible-powerflex/blob/2.4.0/MODULE-LICENSE) for the full terms.
 
 ## Prerequisites
 
 | **Ansible Modules** | **PowerFlex/VxFlex OS Version** | **SDK version** | **Python version** | **Ansible**              |
 |---------------------|-----------------------|-------|--------------------|--------------------------|
-| v2.3.0 |3.6 <br> 4.0 <br> 4.5 | 1.10.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 |
+| v2.4.0 |3.6 <br> 4.0 <br> 4.5 | 1.10.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 |
```
Comment (Jennifer-John) [inline README.md:31]:
```suggestion
| v2.4.0 |3.6 <br> 4.0 <br> 4.5 | 1.11.0 | 3.9.x <br> 3.10.x <br> 3.11.x | 2.14 <br> 2.15 <br> 2.16 |
```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -158,3 +158,8 @@ releases:
         name: resource_group
         namespace: ''
     release_date: '2024-03-29'
+  2.4.0:
+    changes:
+      minor_changes:
+        - Added support for executing Ansible PowerFlex modules and roles on AWS environment
```
Comment (trisha-dell) [inline changelogs/changelog.yaml:164]: Missing full-stop at the end of the statement.

File: docs/Release Notes.md [hunk 4]
```diff
@@ -62,11 +62,11 @@ Limitations
 Distribution
 ------------
 The software package is available for download from the [Ansible Modules
-for PowerFlex GitHub](https://github.com/dell/ansible-powerflex/tree/2.3.0) page.
+for PowerFlex GitHub](https://github.com/dell/ansible-powerflex/tree/2.4.0) page.
```
Comment (trisha-dell) [inline docs/Release Notes.md:65]: Please add the AWS support in the release notes.

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**